Backpack/src/main/res/values/attrs.xml

```diff
@@ -26,14 +26,6 @@
 
   <declare-styleable name="BpkText">
     <attr name="textStyle" format="enum">
-      <enum name="xs" value="0" />
-      <enum name="sm" value="1" />
-      <enum name="base" value="2" />
-      <enum name="lg" value="3" />
-      <enum name="xl" value="4" />
-      <enum name="xxl" value="5" />
-      <enum name="xxxl" value="6" />
-      <enum name="caps" value="7" />
       <enum name="hero1" value="8" />
```

Comment:
Since we removing the old values, would that make sense to use the numbers starting with 0? That'd would make the future changes easier.